src/app.ts

```diff
@@ -109,7 +109,7 @@ console.log('***********************')
 console.log('App directory: ' + RNFS.DocumentDirectoryPath)
 console.log('***********************')
 
-// @ts-expect-error
+// @ts-expect-error - this needs an explanation
```

Comment:
Indeed, it does need an explanation.

This is needed for our logging integration. When we replace `console.log` with our  own custom logging, we use `global.clog` to keep forwarding logs to the normal place.

This is dumb. Instead, our logging infrastructure should use named variable like `export const realLog = console.log`, and then call `realLog()` in the logging system. That avoids polluting the global scope, and makes it clear where this is being used (TypeScript becomes aware of what's going on).